recipes-kernel/linux/linux-stable-common.inc

```diff
@@ -3,7 +3,6 @@ require recipes-kernel/linux/linux-yocto.inc
 
 SRC_URI = "\
     https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-${PV}.tar.xz \
-    file://defconfig \
```

Comment:
This is no need for creating a separate `linux-stable` recipes for the different boards as the only difference is the `defconfig` file.

The directory structure should be enough, e.g:

```
linux-stable/clearfog/defconfig
linux-stable/clearfog-pro/defconfig
``` 

The current `linux-stable_4.19.bb` recipe would pick up the correct file depending on the machine name.

So leave this recipe as is.